aws/provider.go

```diff
@@ -221,6 +221,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_ec2_local_gateway_virtual_interface":        dataSourceAwsEc2LocalGatewayVirtualInterface(),
 			"aws_ec2_local_gateway_virtual_interface_group":  dataSourceAwsEc2LocalGatewayVirtualInterfaceGroup(),
 			"aws_ec2_local_gateway_virtual_interface_groups": dataSourceAwsEc2LocalGatewayVirtualInterfaceGroups(),
+			"aws_ec2_spot_price":                             dataSourceAwsEc2SpotPrice(),
```

Comment:
@p0pr0ck5 @bflad This data source is documented as `aws_ec2_instance_spot_price` (note `instance`) but as this line demonstrates, the name is _actually_ `aws_ec2_spot_price`. So either the docs or the data source name here has to be changed.

Reply:
Please see: #14680